modules/cli/src/main/scala/scala/cli/commands/export0/Export.scala

```diff
@@ -212,13 +226,23 @@ object Export extends ScalaCommand[ExportOptions] {
     }
     else {
       val sbtVersion = options.sbtVersion.getOrElse("1.10.1")
+      // todo: how to use it from deps.sc file here?
```

Comment:
declare it in the `cli` module's `Constants.scala` file via `build.sc`, and then call `Constants` from here.